.github/workflows/continuous_integration.yml

```diff
@@ -13,7 +13,18 @@ defaults:
     shell: bash
 
 jobs:
+  branch_check:
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'push' }}
+    steps:
+      - name: Fail on restricted branch
+        # https://docs.github.com/en/actions/learn-github-actions/expressions#example-matching-an-array-of-strings
+        if: contains(fromJSON('["cloud"]'), github.ref_name)
```

Comment:
Just to be clear, this will block any branch name containing `cloud`

Reply:
I don't think so. According to the [documentation](https://docs.github.com/en/actions/learn-github-actions/expressions#contains), if the search is an array, it will only return true if item is inside search. If search is a string, it returns true if item is a substring

Reply:
Thanks for clarification